externs/d3-3.4.1.js

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable jsdoc/require-returns-check */
```

Comment:
Why were these necessary? We do not lint externs.

Reply:
It's not really necessary, but some of the other externs have it